### DISCUSSION TITLE
2.3.22

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#12061] [General]
*  Remove experimental useWindowsDnsCache config

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/2.3.22'>2.3.22</a>.</em>

**Comment by atif-cloud** [comment]
will it solve this problem  "System has not been booted with systemd as init system (PID 1). Can't operate.
Failed to connect to bus: Host is down"

**Reply by JCarlosQuiroz** [reply]
I think this answers your question -------->   https://learn.microsoft.com/en-us/windows/wsl/systemd

**Comment by astroboylrx** [comment]
Whenever an update is released, Windows Store terminates my WSL and updates it for me, regardless of running processes. Is this designed behavior or is there anything I can do to prevent this? 
I already turned off auto app updates in the Store settings and even enabled "Turn off Automatic Downloads and Install of updates" in the group policy.

WSL version: 2.3.22.0
Kernel version: 5.15.153.1-2
WSLg version: 1.0.65
MSRDC version: 1.2.5620
Direct3D version: 1.611.1-81528511
DXCore version: 10.0.26100.1-240331-1435.ge-release
Windows version: 10.0.22635.4082
